src/main/java/org/jenkinsci/plugins/authorizeproject/AuthorizeProjectProperty.java

```diff
@@ -135,7 +138,13 @@ public Authentication authenticate(Queue.Item item) {
      */
     @Initializer(after = InitMilestone.PLUGINS_STARTED)
     public static void setStrategyCritical() {
-        Items.XSTREAM2.addCriticalField(AuthorizeProjectProperty.class, "strategy");
+        //TODO the reflection could be removed when the addCriticalField will be un-restricted (PR: https://github.com/jenkinsci/jenkins/pull/3066)

```

Comment:
I would consider it as a defect. My proposal would be to convert [JENKINS-47342](https://issues.jenkins-ci.org/browse/JENKINS-47342) to a bug

Reply:
Converted as requested

Reply:
Please create another and more descriptive bug ticket.
No one can get what is the bug from [JENKINS-47342](https://issues.jenkins-ci.org/browse/JENKINS-47342) and what resolves it.